src/shared/components/palette/fixed-list/write-fixed-list-field.component.ts

```diff
@@ -32,4 +32,8 @@ export class WriteFixedListFieldComponent extends AbstractFieldWriteComponent im
     this.fixedListFormControl = this.registerControl(new FormControl(this.caseField.value)) as FormControl;
     this.fixedListFormControl.setValue(this.caseField.value);
   }
+
+  public inputBlur() {
+    this.fixedListFormControl.markAsUntouched();
```

Comment:
Same thing as above. This will always clear out the display of an error on a control when it loses focus. I can't see how this is desirable.

Reply:
same as above